services/workflows-service/src/common/utils/winston-logger/winston-logger.ts

```diff
@@ -5,16 +5,26 @@ export class WinstonLogger implements IAppLogger {
   private logger: TWinstonLogger;
 
   constructor() {
+    const isProduction = process.env.NODE_ENV === 'production';
+
+    const productionFormat = format.combine(format.timestamp(), format.json());
+
+    const developmentFormat = format.combine(
```

Comment:
for development purposes you can use pretty print 
`winston.format.prettyPrint()`

Reply:
ive tried `winston.format.prettyPrint()` before printf, it didnt work well - im not sure why